custom.proto

```diff
@@ -21,6 +21,11 @@ message ProfilingResultSet {
   map<string, sensr_proto.ProfilingResult> algo_nodes = 2;
 }
 
+enum Stage {
+  STAGE_NONE = 0;
+  STAGE_DRIVING = 1;
```

Comment:
@minhup could you add more stages?

Reply:
For now, those two are enough. We can add more later.

Reply:
For simplification, STAGE_NONE -> NONE and STAGE_DRIVING -> DRIVING.

Reply:
Unfortunately we should keep the `STAGE_` prefix because enum in protobuf is plain enum so NONE could conflict with others. Also, the style guide suggests putting the type name as a prefix - [ref](https://protobuf.dev/programming-guides/style/#enums). 
```protobuf
enum FooBar {
  FOO_BAR_UNSPECIFIED = 0;
  FOO_BAR_FIRST_VALUE = 1;
  FOO_BAR_SECOND_VALUE = 2;
}
```

Regarding `CONTROLLED`, I also considered the name but wasn't sure if the other project would also "control" something like moving a trailer or controlling a crane to pick up a container. What do you think?

Reply:
Controlled meant being controlled by motion planner. We only send FoG for controlled objects. I didn't consider the trailer cases yet since we are not doing the project yet so no need to support it now. But we need the controlled stage for building our full stack (perception + control). For the prefix, it's fine to keep it though if it makes teh conflict.

Reply:
The car is being driving doesn't necessary being controlled by motion planner.

Reply:
True, that's a good point. I'll rename it with `STAGE_CONTROLLED`.